sycl/test-e2e/Graph/lit.local.cfg

```diff
@@ -1,4 +1,3 @@
 # https://github.com/intel/llvm/issues/17165
 if 'windows' in config.available_features:
    config.unsupported_features += ['arch-intel_gpu_bmg_g21']
-config.unsupported_features += ['level_zero_v2_adapter']
```

Comment:
V2 doesn't have support for MCL yet, can we move this to `Graph/Update/lit.local.cfg`

Reply:
Right, done.